SU-SWS/stanford_profile
# D8CORE-2960 D8CORE-2961 Update paragraph and field names, with icons

# READY FOR REVIEW

# Summary
- Updated paragraph name, field labels/help text and added icons.

# Need Review By (Date)
- asap

# Urgency
- high

# Steps to Test
1. checkout this branch
1. `drush cim`
1. go to create a basic page
1. verify the "List" and "Teaser" paragraph tools have icons.

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
